server/fs-explorer/index.js

```diff
@@ -0,0 +1,20 @@
+let path = require('path'),
+  recursiveReadDir = require('recursive-readdir');
+
+const SUPPORTED_MEDIA = ['.mkv', '.mp4'];
+
+let scanForMedia = function (root, callback) {
+  let ignoreFunc = function (file, stats) {
+    return !stats.isDirectory() && !SUPPORTED_MEDIA.includes(path.extname(file));
```

Comment:
We should do `path.extname(file).toLowerCase()` while checking `!SUPPORTED_MEDIA.includes(path.extname(file));` since there are cases when extensions are in all uppercase. e.g. `.MKV`, `.MP4`